include/MemoryModel/SVFVariables.h

```diff
@@ -130,8 +130,8 @@ class SVFVar : public GenericPAGNodeTy
         {
             if(const Instruction* inst = SVFUtil::dyn_cast<Instruction>(value))
                 return inst->getParent()->getParent();
-            else if (const Argument* arg = SVFUtil::dyn_cast<Argument>(value))
-                return arg->getParent();
+            else if (const SVFArgument* arg = SVFUtil::getArgument(value))
```

Comment:
getArgument takes long time to search the argument. This should be a simple cast.